gsa/src/gmp/models/__tests__/nvt.js

```diff
@@ -154,6 +154,43 @@ describe('nvt Model tests', () => {
     expect(nvt3.severity).toEqual(9.5);
   });
 
+  test('should parse severity and origin from severities', () => {
+    const nvt1 = Nvt.fromElement({
+      severities: {
+        severity: {
+          score: 94,
+          origin: 'Vendor',
+        },
+      },
```

Comment:
```suggestion
      },
      cvss_base: '7.5'
```

Reply:
We should add something like this to make sure that the cvss_base is ignored and the severity from severities is used. That is how it is supposed to work, right, we only use the cvss_base as a fall back?